yellowbrick/text/tsne.py

```diff
@@ -130,43 +132,51 @@ class TSNEVisualizer(TextVisualizer):
     ax : matplotlib axes
         The axes to plot the figure on.
 
-    decompose : string or None
+    decompose : string or None, default: ``'svd'``
         A preliminary decomposition is often used prior to TSNE to make the
-        projection faster. Specify `"svd"` for sparse data or `"pca"` for
-        dense data. If decompose is None, the original data set will be used.
+        projection faster. Specify ``"svd"`` for sparse data or ``"pca"`` for
+        dense data. If None, the original data set will be used.
 
-    decompose_by : int
+    decompose_by : int, default: 50
         Specify the number of components for preliminary decomposition, by
         default this is 50; the more components, the slower TSNE will be.
 
-    classes : list of strings
+    labels : list of strings
         The names of the classes in the target, used to create a legend.
+        Labels must match names of classes in sorted order.
 
     colors : list or tuple of colors
         Specify the colors for each individual class
 
     colormap : string or matplotlib cmap
         Sequential colormap for continuous target
 
+    random_state : int, RandomState instance or None, optional, default: None
```

Comment:
Because TSNE, PCA, and SVD are all stochastic, had to add random_state to get the tests to pass.